Backend/index.js

```diff
@@ -114,6 +120,28 @@ app.post("/friends", async function (req, res) {
   }
 });
 
+/**
+ * Updates a score associated with the user
+ */
+app.put("/users/score/:id", async function (req, res) {
+  if (!req.params || !req.params.id || !req.body) {
```

Comment:
I would put both the id and the score in req.body

So the route would be `/users/score`

The JSON would be:
`{
      "id": 1234,
       "score": 54
} `